aten/src/ATen/native/native_functions.yaml

```diff
@@ -335,17 +339,20 @@
     CPU: baddbmm_out_cpu
     CUDA: baddbmm_out_cuda
 
-- func: bartlett_window(int window_length, TensorOptions options=[]) -> Tensor
+- func: bartlett_window(int window_length, *, ScalarType dtype=float, Layout layout=strided, Device device=\"cpu\") -> Tensor
```

Comment:
these types and defaults look not-pythonic.  You don't have to change this now, but what's the long term plan?

i.e. ScalarType -> Dtype(?), in python this is "torch.dtype", but "dtype dtype" looks a bit strange.  Maybe we should just write out the full types?  e.g:

`func: bartlett_window(int window_length, *, torch.dtype dtype=None, torch.layout layout=torch.strided, torch.device device=None) -> Tensor`

Although then I guess you should write out `torch.Tensor` which seems annoying.

Thoughts?  CC @zdevito.

Reply:
Keep in mind, this also needs be used in the JIT.